notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4410,6 +4410,56 @@ the desire for vengeance [A soldier tried to kill Deunan to avenge her father ha
 mass infertility [Although not really explored in any way, Seven Elders revealed that they planned to release a virus that would make the human race permanently unable to reproduce.]
 
 
+movie: Casshern (2004)
+======================
+
+:: Title
+Casshern
+
+:: Description
+Casshern (キャシャーン, Kyashān) is a 2004 Japanese tokusatsu superhero film
+adaptation of the anime series of the same name. It was written and directed
+by Kazuaki Kiriya.
+
+Synopsis: A race of artificially made humans wage war on mankind.
+
+:: Date
+2004-04-24
+
+:: References
+https://en.wikipedia.org/wiki/Casshern_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+war [The Neo Sapiens waged war on the Eastern Federation. The Eastern Federation mobilized its military against a rebel region. The story is set in the immediate aftermath of a fifty-year war between two rival world powers: the Eastern Federation and Europa.]
+speculative biotechnology [A main novelty of the film is Dr. Azuma's application of special "Neo Cells" (i.e. stem cells) to regenerate human tissue.] 
+revival long after death [Tetsuya being resurrected by means of "Neo Cells" (i.e. stem cells) constitutes a main novelty of the story.]
+misanthropy [The Neo Human leader Burai waxed with contempt for humanity and sought to exterminate the human race with his formidable robot army.]
```

Comment:
I'll buy it. But as there are now two genetically distinct "races" is this not also something along the lines of a race war, ethnic cleansing, or that sort of thing?

Reply:
I used **genocide** as a minor theme in a different context. Although, I suppose you are hitting on a reverse genocide of some sort. Knocked up to major and added to the motivation accordingly.